src/views/templates/actions/hooks/useVirtualMachineTemplatesActions.tsx

```diff
@@ -79,8 +79,9 @@ const useVirtualMachineTemplatesActions: useVirtualMachineTemplatesActionsProps
     await k8sDelete({
       model: TemplateModel,
       resource: template,
-    });
-    history.push(`/k8s/${lastNamespacePath}/templates`);
+    })
+      .then(() => history.push(`/k8s/${lastNamespacePath}/templates`))
+      .catch((e) => console.error(e));
```

Comment:
If you remove this, the modal will handle the error showing the message on top of submit button

Reply:
The catch block is unnecessary here, of course. But it's a good practice to use it whenever working with promises.

Reply:
Yes. The modal will not handle the expection because it's catched here. So no error message will be prompted

Reply:
@hstastna Can you confirm that you don't want the error message to be shown? I don't feel comfortable lgtm'ing this without resolving this conversation.